api/.eslintrc.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
```

Comment:
This should extend the base typescript eslint ruleset `../.eslintrc-ts.js` like this: https://github.com/reach4help/reach4help/blob/master/map/.eslintrc.js

Reply:
We have to make some changes. I did it but then many strange errors appeared.

Reply:
what were the errors?